Source/OxyPlot/Utilities/TrackerHelper.cs

```diff
@@ -0,0 +1,48 @@
+namespace OxyPlot.Utilities
+{
+    internal static class TrackerHelper
+    {
+        /// <summary>
+        /// Gets the nearest tracker hit.
+        /// </summary>
+        /// <param name="series">The series.</param>
+        /// <param name="point">The point.</param>
+        /// <param name="snap">Snap to points.</param>
+        /// <param name="pointsOnly">Check points only (no interpolation).</param>
+        /// <param name="firesDistance">The distance from the series at which the tracker fires</param>
+        /// <returns>A tracker hit result.</returns>
+        public static TrackerHitResult GetNearestHit(
+            Series.Series series, ScreenPoint point, bool snap, bool pointsOnly, double firesDistance)
+        {
+            if (series == null)
+            {
+                return null;
+            }
+
+            // Check data points only
+            if (snap || pointsOnly)
+            {
+                var result = series.GetNearestPoint(point, false);
+                if (IsTrackerOpen(result, point, firesDistance))
+                {
+                    return result;
+                }
+            }
+
+            // Check between data points (if possible)
+            if (!pointsOnly)
+            {
+                var result = series.GetNearestPoint(point, true);
+                if (IsTrackerOpen(result, point, firesDistance))
```

Comment:
Since it is breaking (and it's pretty jarring when you are used to the old behaviour), I think we should make the new behaviour optional, though I can't immediately think of a sensible property/parameter name.

Reply:
Perhaps we can make an additional property and mark it as `Obsolete`? I don't know to which version then keep it

But I am more for the fact that this is the wrong behavior

Reply:
After mulling this over for a while, I think it might be best to provide a property that enables/disables the distance check within `Delta` for _both_ hit tests.
 - It means the beaviour is consistent for interpolated/non-interpolated plots
 - The ability to easily track a series once you've selected it is useful (it's familiar, and useful if you have a messy plot, e.g. I sometimes have to follow a single line out of 200 or so, and letting the tracker do the work for me makes life much better)
 - Users can still use the modifier keys if they want non-snapping or whatever behaviour

I would expect the default to be to disable the distance checks by default, at least for 2.1. I don't think marking the property Obsolete is a good idea, because their behaviour may be desirable.

Reply:
Did I understand correctly that it is better to remove the obsolete mark? Good. The only thing left to do is to make sure the property naming and documentation is well interpreted :)

Reply:
Yes, for now I think it's probably best to stick with what you have working, checking that the default behaviour is the same as the old behaviour. We can make additional changes in a separate PR. Would it be a problem for you if this wasn't in the - somewhat late (my fault) - 2.1 release? Because it is adding additional behaviour one way or another, we probably shouldn't include it. (It's also a feature that can be easily implemented without changes to the core library, so the pre-release implementation and examples should help anyone trying to do something similar).

Reply:
No, that won't be a problem. I figured out how to do a workaround by describing custom class for the manipulator. Thanks for help!